abstract.md

```diff
@@ -1,3 +1,7 @@
+<details> 
```

Comment:
Are there other specs with the abstract in a `details`/`summary`? 

I'd prefer keeping it visible at all times, let's reduce and simplify the text instead.